packages/now-cli/src/util/dev/server.ts

```diff
@@ -1706,7 +1710,24 @@ export default class DevServer {
 
     this.output.debug(`Spawning dev command: ${command}`);
 
-    const p = spawnCommand(command, { stdio: 'inherit', cwd, env });
+    const devPort = new URL(this.address).port;
+    const proxyPort = new RegExp(port.toString(), 'g');
```

Comment:
Adding boundary delimiter might be a good idea:

```suggestion
    const proxyPort = new RegExp(`\\b${port}\\b`, 'g');
```

Reply:
I tried this with CRA which is highlighting the port and it didn't work. 

![image](https://user-images.githubusercontent.com/229881/76458582-22a7b980-63b1-11ea-8f9a-4584ffcec204.png)